synapse/api/auth.py

```diff
@@ -810,6 +810,10 @@ def validate_macaroon(self, macaroon, type_string, verify_expiry, user_id):
         else:
             v.satisfy_general(lambda c: c.startswith("time < "))
```

Comment:
But leave the ``v.satisfy_general(lambda c: c.startswith("time < "))`` so that existing tokens will still work.